winfsp-sys/build.rs

```diff
@@ -27,59 +35,89 @@ fn system() -> String {
         _ => panic!("unexpected install directory"),
     };
 
-    println!("cargo:rustc-link-search={}/lib", directory);
+    println!("cargo:rustc-link-search={}{}", directory, dir);
```

Comment:
When debugging this, I discovered that `directory` already had a slash at the end